Library/Formula/ack.rb

```diff
@@ -1,4 +1,5 @@
 class Ack < Formula
+  desc "A search tool like grep, but optimized for programmers"
   homepage "http://beyondgrep.com/"
```

Comment:
I first was in favor of “homepage then desc then the rest” but I read @adzenith’s comment and now agree with the “description first, then homepage, then the rest”. The `desc` is indeed the more relevant here, and the homepage’s URL doesn’t usually give you much information about the software (say the formula is “foobar”, the URL will likely be `foobar.sourceforge.net` or `some-academic-website.edu/~doe/software/foobar`) and you can open it quickly in your browser with `brew home <formula>`.